frontend/ui/data/creoptions.lua

```diff
@@ -56,6 +58,12 @@ local CreOptions = {
                 args = {0, 48, 96, 167, 212, 300},
                 event = "SetRenderDPI",
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Sets the DPI used to scale CSS absolute units and images:
+- off: ignore absolute units (old engine behaviour).
```

Comment:
Aren't we generally more American (=behavior)? Doesn't matter too much one way or the other.